factory/pom.xml

```diff
@@ -61,266 +65,74 @@
     <url>http://www.google.com</url>
   </organization>
 
-  <developers>
-    <developer>
-      <id>gk5885</id>
-      <name>Gregory Kick</name>
-      <email>gk5885@gmail.com</email>
-      <roles>
-        <role>owner</role>
-        <role>developer</role>
-      </roles>
-      <timezone>-6</timezone>
-    </developer>
-    <developer>
-      <id>eamonnmcmanus</id>
-      <name>Éamonn McManus</name>
-      <email>emcmanus@google.com</email>
-      <organization>Google</organization>
-      <organizationUrl>http://www.google.com</organizationUrl>
-      <roles>
-        <role>owner</role>
-        <role>developer</role>
-      </roles>
-      <timezone>-8</timezone>
-    </developer>
-  </developers>
+  <modules>
+    <module>annotations</module>
+    <module>processor</module>
+  </modules>
+
+  <dependencyManagement>
```

Comment:
It looks like probably few if any of these are used by `annotations`—which looks like it might be the only `pom.xml` that uses `auto-factory-aggregator` as its parent.

- Should the processor also use `auto-factory-aggregator` as its parent?
- If not, do we need any of these dependencies here? (And could the _plugin_ configuration be pushed down into the `annotations` build?)
- If so, might it still make sense to push the dependencies down into `processor` if that's the only place that they're used?

Basically, I can understand emptying the aggregator our (as was done in https://github.com/google/auto/pull/889/files, which I see we've also neglected...) and putting everything that each subproject needs in that subproject, and I can understand putting the intersection of what the two subprojects need into the aggregator, and I can probably understand putting the _union_ of what the two subprojects need into the aggregator. Currently, though, we use the aggregator to configure only for `annotations`, and we probably configure more than we need.